matrix-vis.js

```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
+        var m = " ";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            Type=JSON.stringify(nodes.get(params.nodes[0]).blob.type, undefined, 2);
+            Sender=JSON.stringify(nodes.get(params.nodes[0]).blob.sender, undefined, 2);
+            t1="Event Type : "+Type;
+            t2="Creator : "+Sender;
+            if (Type==JSON.stringify("m.room.member")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.membership, undefined, 2);
+            t3="Action : "+m;
+            }
+            else if (Type==JSON.stringify("m.room.message")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.body, undefined, 2);
+            t3="Message : "+m;
+            }
         }
-        document.getElementById('eventInfo').innerHTML = text;
+        document.getElementById('eventInfo').innerHTML = t1 +"<br>"+ t2 +"<br>"+ t3;
```

Comment:
It would be better to store `t1 t2 t3` in an array and then insert the `<br>` programmatically via `array.join("<br>")`. That way you won't need to keep updating this line every time you decide to add a new line to the output.